index.html

```diff
@@ -33,6 +33,293 @@
+      <a data-cite="screen-capture#dfn-monitor">monitors</a>
+      using the
+      <code>navigator.mediaDevices.getAllScreenMedia</code>
+      method defined in this document.
+    </p>
+    <pre class="example highlight">
+      try {
+        const mediaStreams = await navigator.mediaDevices.getAllScreenMedia();
+        mediaStreams.forEach((mediaStream, index) => {
+          videoElements[index].srcObject = mediaStream;
+        })
+      } catch (e) {
+        console.log('Unable to acquire screen captures: ' + e);
+      }
+    </pre>
+  </section>
+  <section id="terminology">
+    <h2>
+      Terminology
+    </h2>
+    <p>
+      This document uses the definition of {{MediaStream}}, {{MediaStreamTrack}}
+      from [[!GETUSERMEDIA]], <a data-cite="url#concept-url-origin">origin</a> from [[!url]],
+      {{ScreenDetailed}} from [[!window-placement]],
+      and <a data-cite="screen-capture#dfn-display-surface">monitor</a>
+      from [[!screen-capture]].
+    </p>
+  </section>
+  <section id="capture_multiple_display_surfaces">
+    <h2>
+      Capturing multiple monitors
+    </h2>
+    <p>
+      Capture of all <a data-cite="screen-capture#dfn-monitor">monitors</a>
+      is enabled through the addition of a new
+      {{MediaDevices/getAllScreenMedia}} method on the {{MediaDevices}}
+      interface.
+    </p>
+    <section>
+      <h2>
+        <dfn>MediaDevices</dfn> Additions
+      </h2>
+      <pre class="idl">
+          partial interface MediaDevices {
+            Promise&lt;sequence&lt;MediaStream&gt;&gt; getAllScreenMedia();
+          };
+        </pre>
+      <dl data-link-for="MediaDevices" data-dfn-for="MediaDevices" class="methods">
+        <dt>
+          <dfn>getAllScreenMedia</dfn>
+        </dt>
+        <dd>
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> MUST return a list of {{MediaStream}}
+            objects, where each stream belongs to exactly one of the
+            <a data-cite="screen-capture#dfn-monitor">monitors</a> attached to the device.
+          </p>
+          <p>
+            {{PermissionState/"granted"}} permissions cannot be persisted.
+          </p>
+          <p>When the {{MediaDevices/getAllScreenMedia()}}
+            method is called, the <a href="#dfn-user-agent">user agent</a> MUST run the following
+            steps:</p>
+          <ol>
+            <li>
+              <p>Let <var>p</var> be a new promise.</p>
+            </li>
+            <li>
+              <p>Run the following steps in parallel:</p>
+              <ol>
+                <li>
+                  <p>Optionally, e.g., for security reasons, or due to platform
+                    limitations, jump to the step labeled <em>Permission Failure</em> below.</p>
+                </li>
+                <li>
+                  <p>The <a href="#dfn-user-agent">user agent</a> MUST obtain permission
+                  by appropriate means, e.g. by asking the user for permission or by
```

Comment:
do not allow asking for user permission ("user agent must return with permission denied").

Reply:
What do we intend to do on platforms other than CrOS, where Chrome might not have been running as the user logged in? On those devices, the user won't have seen a warning before they began interacting with the device.

Reply:
i added a paragraph in privacy indicator requirements on this as well --> the user must be provided with the means to look up whether getAllScreenMedia is enabled.